proposals/lip-0057.md

```diff
@@ -1470,6 +1475,7 @@ def unbanValidator(address: Address)-> None:
         raise Exception('The specified validator is not banned.')
     if len(validatorStore(address).reportMisbehaviorHeights) < REPORT_MISBEHAVIOR_LIMIT_BANNED:
```

Comment:
I find it a bit odd then in the case of `len(validatorStore(address).reportMisbehaviorHeights) >= REPORT_MISBEHAVIOR_LIMIT_BANNED` we just do not do anything. I would find it better if for all cases that do not allow unbanning, an exception is thrown and in the success case we unband and update the weight.

Reply:
I updated the function according to your proposal.